src/components/ai_models_list/AiModelList.tsx

```diff
@@ -46,55 +48,70 @@ export default function AiModelList(props: AiModelListProps): React.JSX.Element
 
   const getValidationState = () => validation?.aiModels
 
+  const button = (
```

Comment:
Should we pull this button into a common class and use it for all the components?